app/systems/[system]/page.tsx

```diff
@@ -194,9 +388,27 @@ function SendShipMenu({ waypoint }: { waypoint: string }) {
         <Button variant="outline">Send Ship</Button>
       </DropdownMenuTrigger>
       <DropdownMenuContent className="w-56">
-        <SendShipGroup shipGroup={command} groupName="COMMAND" destinationWaypoint={waypoint} />
-        <SendShipGroup shipGroup={surveyors} groupName="SURVEYOR" destinationWaypoint={waypoint}/>
-        <SendShipGroup shipGroup={excavators} groupName="EXCAVATOR" destinationWaypoint={waypoint}/>
+        {command.length > 0 && (
```

Comment:
I made it so if you don't own a ship of a particular type (ex: SURVEYOR) then that doesn't show up in the list.  Eventually I want to exclude any ships that are already at that waypoint from the list(s) too.